src/utilities/activity-utils.ts

```diff
@@ -232,7 +239,6 @@ export const getAllUrlsInActivity = async (activity: Activity, urls: string[] =
       if (authorData?.s3Url) {
         const response = await fetch(authorData.s3Url);
         const glossaryJson = await response.json();
-        urls.push(authorData.s3Url);
```

Comment:
This was removed as it is now being picked up with the other urls in the author_data